### ISSUE TITLE
关于训练的具体参数问题

### ISSUE THREAD

**Issue by 15071230989** [ShawnBIT/AI-Challenger-Retinal-Edema-Segmentation#6]
您好！
我使用了您的unet++以及wce+eldice+bce损失函数
得到的结果与您ppt上的结果有些差距

我使用优化器是SGD, momentum=0.9， weight decay = 5e-4
batchsize = 4， epochs=40， learning rate =0.01使用poly策略
预处理使用了3d stack和规范化

使用验证集的得到的结果为rea：0.8033
                                             srf：0.7829
                                            ped：0.7357
您的结果是rea：0.755
                   srf：0.859
                 ped：0.784

请问有能否给出一些具体训练方案的建议

**Comment by ShawnBIT** [comment]
首先，对学习过程影响最大的超参数就是learning rate，我当时用的是1e-3，你现在是1e-2，应该影响比较大。其次你的batchsize比较小，我当时用的4卡，batchsize是16。最后像Adam和step decay策略应该也会有些微弱的影响。总之，如果你希望复现我的结果，建议你用我给的超参数设置。同时我也放了我训练出的model checkpoint，你也可以下载去测试。

**Comment by 15071230989** [comment]
好的，非常感谢！
